jest-sa11y-setup.js

```diff
@@ -0,0 +1,6 @@
+// Since this is not a typescript project, disabling typescript eslint warning
+// TODO (lint): using 'exclude: ["./packages/test-integration/**/*.js"]' in tsconfig.eslint.json doesn't work
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { registerSa11yMatcher } = require('@sa11y/jest');
```

Comment:
@albarivas If the project supports ES6 imports (looks like this one does), you can simply use 
`import { registerSa11yMatcher } from '@sa11y/jest';`

That would remove the need for the 3 lines above - comment, TODO and eslint directive.
I am sorry if the docs were not clear - I have [updated the docs](https://github.com/salesforce/sa11y/pull/39/commits/759fd734fa8f6d3c68e34fa58c1a95edda11e2d3) to indicate that either CommonJS or ES6 imports can be used.

Functionally they might be equivalent - but since your repos would serve as example to others just wanted to clarify.

Reply:
@albarivas I'd suggest we use the ES6 `import` statement. Thanks @mohanraj-r.